requirements-hpu-1.18.txt

```diff
@@ -0,0 +1,12 @@
+# Common dependencies
```

Comment:
Lets not add this, there is description in issues and release notes on how to handle this issue.